### DISCUSSION TITLE
Comprehensive and representative examples

### DISCUSSION THREAD

**Discussion by lucasaarcoverde** [vtex/shoreline#732] [General]
> ℹ️ This was originally opened by @matheusps on **14 jan 2021**

**Is your feature request related to a problem? Please describe.**
The admin-ui is lacking a folder with common real-world examples. [Benchmark](https://github.com/vercel/next.js/tree/canary/examples).

This would help users to better understand how to get started with the library.

**Initial use cases:**
- IO Application
- CRA
- Gatsby
- Next.js

**Comment by davicostalf** [comment]
Comentários do CSAT de engenharia em 2023 H1:

> We need to investigate the code to get full possibilities and parameters for the components because the example page doesn't show all the possibilities and has a bad experience to copy code there.

> A documentação é boa. Sugiro incluir mais detalhes e exemplos nos componentes que podem utilizar chamadas async, uma atenção redobrada nas breaking changes (deixar claro o impacto dessas mudanças), aos componentes obsoletos e a eventuais tokens que não funcionam corretamente.

> Gosto da doc, mas poderia ser melhor e um pouco mais demonstrativa, já aconteceu de algum componente ser lançado e ter sua explicação, mas na implementação era possível perceber que faltava algo, então é necessário ir debugando pelo código do componente para entendê-lo bem

> às vezes falta alguns exemplos de caso de uso, e sempre tenho problema ao copiar o código dos exemplos
